pkg/components/velero/component_test.go

```diff
@@ -195,3 +196,49 @@ component "velero" {
 		t.Fatalf("Rendered manifests shouldn't be empty")
 	}
 }
+
+func TestRenderManifestResticToleration(t *testing.T) {
+	configHCL := `
+component "velero" {
+  provider = "restic"
+  restic {
+    credentials = "foo"
+
+    backup_storage_location {
+      bucket   = "foo"
+      provider = "aws"
+    }
+		
+    tolerations {
+      key = "TestResticToletrationKey"
+      value = "TestResticToletrationValue"
+      operator = "Equal"
+      effect = "NoSchedule"
+      toleration_seconds = "1"
+    }
+  }
+}
+`
+
+	component := NewConfig()
+
+	body, d := util.GetComponentBody(configHCL, Name)
+	if d != nil {
```

Comment:
`d` is type `hcl.Diagnostics`, we should use `HasErrors()` instead of `!=  nil`.